managers/templates/managers/project_update.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+{% block content %}
+{% load i18n %}
+{% load rest_framework %}
+{% load static %}
+<head>
```

Comment:
This should be put in `{% block extra_head %}` which should be defined in `base.html` at the end of `<head>` tag.

Reply:
Done